### ISSUE TITLE
Loading a model with CategoryEncoding layer from keras

### ISSUE THREAD

**Issue by hhadzem** [deepjavalibrary/djl#1542]
## Description

As far as I can tell, using a String as a feature is only supported by implementing the Translator and processing the input string in processInput method. 
I have a saved model from TF python, which already has preprocessing layers embedded (tf.keras.layers.Normalize and tf.keras.layers.CategoryEncoding). 

Is it possible to load the model including those layers and using String as a feature as of now, and if not, do you plan on implementing it?
